### DISCUSSION TITLE
How to search for deleted content item?

### DISCUSSION THREAD

**Discussion by dodyg** [OrchardCMS/OrchardCore#11519] [Q&A]
Is there any way to do it from the web interface?

The problem is I can't even re-enter the content because the system complaints about the path already exists.

**Comment by Skrypt** [comment]
Look in your AutorouteIndex table there is probably a new content item that uses the same path.

**Reply by dodyg** [reply]
* Content A is deleted. I can't find it. It has path /en/content-a
* Now I am stuck. I want to keep the path /en/content-a
* I want to find Content A otherwise I will lose the path.

**Reply by Skrypt** [reply]
When we delete a content item generally the AutoroutePart is also removed or unpublished. Need to verify.

**Reply by Skrypt** [reply]
For searching deleted content items with Lucene we can't right now. They are not indexed. You need to search by using the ContentItemIndex table. It would require a specific index that contains only deleted content items to not temper with the actual frontend search results.

**Reply by ns8482e** [reply]
Look for IsLatest = false and IsPublished = false

**Reply by dodyg** [reply]
The problem is that ContentItemIndex doesn't really have any information regarding the status of the content. It has `Published`. That's it.

I solved this problem by going to AutorouteIndex table directly in SQLite, find the path and rename that path manually with some random path. Then I create a new content to occupy the desired path.

**Reply by dodyg** [reply]
`SELECT * FROM ContentItemIndex Where Latest = false and Published = false And DisplayText like 'project%'` works in finding the content in ContentItemIndex

**Reply by ns8482e** [reply]
Btw, OC should delete  IsLatest = false and IsPublished = false

**Reply by dodyg** [reply]
I think we are missing a Trash functionality. Give people the option to restore or empty the trash.
